apps/web/src/components/RecruitmentAvailabilityDialog/RecruitmentAvailabilityDialog.tsx

```diff
@@ -150,7 +150,7 @@ const RecruitmentAvailabilityDialog = ({
               <p data-h2-margin-bottom="base(x1)">
                 {intl.formatMessage({
                   defaultMessage:
-                    "If you've recently taken a job or simply no longer want to be considered for opportunities, you can disable your availability in this recruitment process. <strong>This will <emphasize>not</emphasize> remove you from the recruitment</strong> itself and you can always re-enable your availability if you change your mind.",
+                    "If you've recently taken a job or simply no longer want to be considered for opportunities, you can disable your availability in this recruitment process. <strong>This will <emphasize>not</emphasize> remove you from the process</strong> itself and you can always re-enable your availability if you change your mind.",
                   id: "3AwFyW",
```

Comment:
This `id` needs to be regenerated when anything changes in the object (REF: https://github.com/GCTC-NTGC/gc-digital-talent/actions/runs/8286743739/job/22677461691?pr=9759#step:9:155)

The French string's `id` will need to change and `defaultMessage` to reflect the copy change.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/3d2d51b3cd328564e7707e0e4879939b85341331/apps/web/src/lang/fr.json#L906-L909

```suggestion
                  id: "rR252Q",
```

Reply:
Not sure how I managed to not update the id 😕